backend/trafficrules/src/main/java/com/coffeejug/trafficrules/controller/StatisticController.java

```diff
@@ -0,0 +1,87 @@
+package com.coffeejug.trafficrules.controller;
+
+import com.coffeejug.trafficrules.exception.BadRequestException;
+import com.coffeejug.trafficrules.service.StatisticService;
+import com.coffeejug.trafficrules.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.repository.query.Param;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/api/v1/statistics")
+public class StatisticController {
+
+    private final StatisticService statisticService;
+    private final UserService userService;
+    private static final List<String> allowedTypes = Arrays.asList(

```

Comment:
Please follow Java naming convention for constant names

Reply:
But if we implement it in a way that the endpoint will return the entire statistic then we don't need it at all and in this way the code will become more readable